pkg/e2e/operators/operators.go

```diff
@@ -474,7 +474,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s configMap to exist", (timeoutDuration - elapsed), operatorLockFile)
+				log.Printf("Waiting %v for %s configMap to exist", timeoutDuration-elapsed, operatorLockFile)
```

Comment:
Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.